test/data/html/tests.js

```diff
@@ -300,6 +300,12 @@ exports.test_data = {
             ],
             indent_attr: ' ',
             indent_over80: ' '
+        }, {
+            options: [
+                { name: "wrap_attributes", value: "'force-aligned'" }
+            ],
+            indent_attr: '\\n     ', /* Number of spaces should be dynamic */
```

Comment:
add more matrix vars to match the dynamic size and adjust the tests?  Specifically consider changing the elements used for the tests to ones that will align in the same position.  Or one of a couple positions...